src/frontend/js/components/CourseGlimpseList/index.tsx

```diff
@@ -27,7 +33,19 @@ export const CourseGlimpseList = ({
 }: CourseGlimpseListProps & CommonDataProps) => {
   return (
     <div className="course-glimpse-list">
-      <div className="course-glimpse-list__count">
+      <div
+        className="offscreen course-glimpse-list__screen-reader-count"
```

Comment:
It seems the class `course-glimpse-list__screen-reader-count` is for test purpose only. So if I'm right, you should prefer to use `data-testid`.

https://testing-library.com/docs/queries/bytestid/

Reply:
Oh yes!